lib/mayaUsd/sceneIndex/proxyShapeSceneIndexPlugin.cpp

```diff
@@ -110,6 +147,33 @@ MayaUsdProxyShapeSceneIndex::MayaUsdProxyShapeSceneIndex(
 
 MayaUsdProxyShapeSceneIndex::~MayaUsdProxyShapeSceneIndex() { }
 
+MayaUsdProxyShapeSceneIndexRefPtr MayaUsdProxyShapeSceneIndex::New(
+    MayaUsdProxyShapeBase*                 proxyShape,
+    const HdFlatteningSceneIndexRefPtr&    flatteningSceneIndex,
+    const UsdImagingStageSceneIndexRefPtr& usdImagingStageSceneIndex)
+{
+    // Create the proxy shape scene index which populates the stage
+    return TfCreateRefPtr(new MayaUsdProxyShapeSceneIndex(
+        flatteningSceneIndex, usdImagingStageSceneIndex, proxyShape));
+}
+
+Ufe::Path MayaUsdProxyShapeSceneIndex::InterpretRprimPath(
+    const HdSceneIndexBaseRefPtr& sceneIndex,
+    const SdfPath&                path)
+{
+    if (MayaUsdProxyShapeSceneIndexRefPtr proxyShapeSceneIndex
+        = TfStatic_cast<MayaUsdProxyShapeSceneIndexRefPtr>(sceneIndex)) {
+        MStatus  status;
+        MDagPath dagPath(
+            MDagPath::getAPathTo(proxyShapeSceneIndex->_proxyShape->thisMObject(), &status));
+        return Ufe::Path({ MayaUsd::ufe::dagPathToPathSegment(dagPath),
+                           // Disable UfePath instances, passing in -1 to indicate this
+                           MayaUsd::ufe::usdPathToUfePathSegment(path, -1) });
```

Comment:
All instances is the default, please remove the -1 argument (especially if you're not using the proper defined constant).